monarchmoney/monarchmoney.py

```diff
@@ -1590,6 +1589,61 @@ async def get_transaction_tags(self) -> Dict[str, Any]:
             operation="GetHouseholdTransactionTags", graphql_query=query
         )
 
+    async def set_transaction_tags(
+        self,
+        transaction_id: str,
+        tag_ids: List[str],
+    ):
```

Comment:
Set the return type on the function

```suggestion
    ) -> Dict[str, Any]:
```